_data/sites.json

```
@@ -14960,6 +14960,19 @@
         ]
     },
 
+    {
+        "name": "OpenPhone",
+        "url": "https://support.openphone.com/hc/en-us/requests/new",
+        "email": "help@openphone.co",
+        "email_subject": "Account deletion request",
+        "email_body": "I request a deletion of my OpenPhone account associated with this email address.",
+        "difficulty": "medium",
```

```suggestion
        "difficulty": "hard",
```
If it needs support contact we classify as hard